app/views/equipment_items/index.html.erb

```diff
@@ -48,7 +48,6 @@
   <% else %>
     <%= link_to "New Equipment Item", new_equipment_item_path, class: 'btn btn-primary' %>
     <%= link_to "Import Equipment Item", equip_import_page_path, class: 'btn btn-default' %>
-    <%= link_to "Export Equipment Data", categories_path(format: 'zip'), :class => "btn btn-default" %>
```

Comment:
So we don't want to include the export link on the equipment items index? Any reason why not?